packages/main/src/TimePickerClock.ts

```diff
@@ -704,7 +704,7 @@ class TimePickerClock extends UI5Element {
 				this._movSelectedValue = 0 + this._selectedValue;
 			}
 		} else if (evt.type === "mousemove") {
-			if (!this._dimensionParameters.radius) {
+			if (!this._dimensionParameters) {
```

Comment:
we have to change also property declaration and to change `!` to `?` since with version 2.0 the properties are not explicitly initialized based on the type they have. we need to use property initialzer if we want them to have default values.